app/src/main/java/org/wikipedia/bridge/JavaScriptActionHandler.kt

```diff
@@ -142,7 +142,7 @@ object JavaScriptActionHandler {
     fun mobileWebChromeShim(): String {
         return "(function() {" +
                 "let style = document.createElement('style');" +
-                "style.innerHTML = '.header-chrome { visibility: hidden; margin-top: 48px; height: 0px; } #page-secondary-actions { display: none; } .mw-footer { margin-bottom: 48px; }';" +
+                "style.innerHTML = '.header-chrome { visibility: hidden; margin-top: 48px; height: 0px; } #page-secondary-actions { display: none; } .mw-footer { margin-bottom: 48px; } .page-actions-menu { display: none; } .minerva__tab-container { display: none; }';" +
```

Comment:
Would it be possible to update `padding-bottom` in `.minerva-footer` so that the bottom toolbar will not cover the footer?

Reply:
yep